Pipfile

```diff
@@ -61,26 +60,29 @@ snekbox = "gunicorn -c config/gunicorn.conf.py snekbox.api.app"
 devsh = "sh scripts/dev.sh"
 build = """
     docker build \
-        -t pythondiscord/snekbox:latest \
+        -t ghcr.io/python-discord/snekbox:latest \
         -f docker/Dockerfile \
         .
 """
 buildbase = """
     docker build \
-        -t pythondiscord/snekbox-base:latest \
-        -f docker/base.Dockerfile \
+        -t ghcr.io/python-discord/snekbox-base:latest \
+        -f docker/Dockerfile \
+        --target base \
         .
 """
 buildvenv = """
     docker build \
-        -t pythondiscord/snekbox-venv:latest \
-        -f docker/venv.Dockerfile \
+        -t ghcr.io/python-discord/snekbox-venv:latest \
+        -f docker/Dockerfile \
+        --target venv \
         .
 """
 builddev = """
     docker build \
-        -t pythondiscord/snekbox-venv:dev \
-        -f docker/venv.Dockerfile \
+        -t ghcr.io/python-discord/snekbox-venv:dev \
+        -f docker/Dockerfile \
+        --target venv \
         --build-arg DEV=1 \
         .
 """
```

Comment:
I don't think the additional scripts still have a use given that it's a single image now.

Reply:
`builddev` may be useful, but probably not since `dev.sh --build` and `testb` exist.